hazelcast-client/src/main/java/com/hazelcast/client/config/XmlClientConfigBuilder.java

```diff
@@ -131,6 +131,7 @@ public XmlClientConfigBuilder() {
     protected Document parse(InputStream inputStream) throws Exception {
         DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
         dbf.setNamespaceAware(true);
+        dbf.setFeature("http://apache.org/xml/features/disallow-doctype-decl", true);
```

Comment:
I believe this is a Xerces 2 -specific feature. Other parsers will probably ignore it. 
Anyway, we accept configuration input from trusted sources only. So I think this is ok.